pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -562,8 +566,8 @@ func checkForUpgrade(cv *configv1.ClusterVersion) error {
 	if c := findClusterOperatorStatusCondition(cv.Status.Conditions, "Invalid"); c != nil && c.Status == configv1.ConditionTrue {
 		results = append(results, fmt.Sprintf("the cluster version object is invalid, you must correct the invalid state first:\n\n  Reason: %s\n  Message: %s\n\n", c.Reason, strings.ReplaceAll(c.Message, "\n", "\n  ")))
 	}
-	if c := findClusterOperatorStatusCondition(cv.Status.Conditions, configv1.OperatorDegraded); c != nil && c.Status == configv1.ConditionTrue {
-		results = append(results, fmt.Sprintf("the cluster is experiencing an upgrade-blocking error:\n\n  Reason: %s\n  Message: %s\n\n", c.Reason, strings.ReplaceAll(c.Message, "\n", "\n  ")))
+	if c := findClusterOperatorStatusCondition(cv.Status.Conditions, clusterStatusFailing); c != nil && c.Status == configv1.ConditionTrue {
+		results = append(results, fmt.Sprintf("the cluster is experiencing a possibly-upgrade-blocking error:\n\n  Reason: %s\n  Message: %s\n\n", c.Reason, strings.ReplaceAll(c.Message, "\n", "\n  ")))
```

Comment:
instead of `The cluster is experiencing a possibly-upgrade-blocking error` what about `Here are the errors on the cluster which might impact upgrades`

Reply:
I've pushed d0030fcfd where I use `the cluster is experiencing an error reconciling ${CURRENT_TARGET_VERSION}`.  And it falls back to the target pullspec in the unlikely event of not knowing the version.